helmfile/charts/node-local-dns/templates/node-local-dns.yaml

```diff
@@ -17,6 +17,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: node-local-dns
+  namespace: kube-system
```

Comment:
We should not add namespaces like this in the helm charts. Instead we let helm add the namespace
```suggestion
```
Same for the rest of the files

Reply:
https://github.com/elastisys/compliantkubernetes-apps/blob/74788e09b707313a6f2e9bc2ff2fa3815c385279/bootstrap/namespaces/helmfile/values/namespaces-sc.yaml.gotmpl should be enough right?